### PR TITLE
Added documentation for review module

### PR DESCRIPTION
**Pull request by geektutor**
### Related issues

closes #

### Which environment this relates to
Check your case. In case of any doubts please read about [Release Cycle](https://docs.vuestorefront.io/guide/basics/release-cycle.html)

- [ ] Test version (https://test.storefrontcloud.io) - this is a new feature or improvement for Vue Storefront. I've created branch from `develop` branch and want to merge it back to `develop`
- [ ] RC version (https://next.storefrontcloud.io) - this is a stabilisation fix for Release Candidate of Vue Storefront. I've created branch from `release` branch and want to merge it back to `release`
- [ ] Stable version (https://demo.storefrontcloud.io) - this is an important fix for current stable version. I've created branch from `hotfix` or `master` branch and want to merge it back to `hotfix`

### Upgrade Notes and Changelog

- [x] No upgrade steps required (100% backward compatibility and no breaking changes)
- [ ] I've updated the [Upgrade notes](https://github.com/DivanteLtd/vue-storefront/blob/develop/doc/Upgrade%20notes.md) and [Changelog](https://github.com/DivanteLtd/vue-storefront/blob/develop/CHANGELOG.md) on how to port existing VS sites with this new feature

**IMPORTANT NOTICE** - Remember to update `CHANGELOG.md` with description of your change

### Contribution and currently important rules acceptance

- [x] I read and followed [contribution rules](https://github.com/DivanteLtd/vue-storefront/blob/master/CONTRIBUTING.md)
- [x] I read the [TypeScript Action Plan](https://github.com/DivanteLtd/vue-storefront/blob/master/doc/TypeScript%20Action%20Plan.md) and adjusted my PR according to it
- [x] I read about [Vue Storefront Modules](https://github.com/DivanteLtd/vue-storefront/blob/master/doc/api-modules/about-modules.md) and I am aware that every new feature should be a module

### CODE REVIEW

File: docs/guide/modules/review.md [hunk 1]
```diff
@@ -1,200 +1,45 @@
-# Review module
+# Cart module
```
Comment (kkdg) [inline docs/guide/modules/review.md]: Cart should be changed to Review, right?

**Comment by patzick** [comment]
@kkdg could you re-review this one?

**Comment by kkdg** [comment]
OK, i will finish it today.

File: docs/guide/modules/review.md [hunk 2]
```diff
@@ -1,200 +1,45 @@
...
 
 **Methods**
 
-- `applyCoupon(code)` - applies cart coupon
-- `removeCoupon` - removes currently applied cart coupon
-- `toggleMicrocart` - open/close microcart
+- `addReview(review)` - adds review to the product.
 
-### MicrocartButton
+### Review
```
Comment (kkdg) [inline docs/guide/modules/review.md]: It should be `Reviews`, because it fetches an array of `Review`

File: docs/guide/modules/review.md [hunk 3]
```diff
@@ -1,200 +1,45 @@
...
-- `cartSavedAt` - (int) timestamp for the latest cart - localForage (local browser) state sync,
-- `bypassToAnon` - (bool) whenever there is a client's quote lock on the cart (for example the cart is currently in sync with Magento etc.) we're using the guest cart for current order; in that case this value is set to true,
-- `shipping` - (object) currently selected shipping method - only when NOT using `cart.synchronize_totals` (if so, the shipping and payment's data comes from Magento2),
-- `payment` - (object) currently selected shipping method - only when NOT using `cart.synchronize_totals` (if so, the shipping and payment's data comes from Magento2),
-- `cartItems` - collection of the cart items; the item format is the same as described in [ElasticSearch Data formats](https://github.com/DivanteLtd/vue-storefront/blob/master/doc/ElasticSearch%20data%20formats.md) - the `product` class; the only difference is that the (int) `qty` field is added
+- `items` - reviews
 
 ### Events
 
 The following events are published from `cart` store:
```
Comment (kkdg) [inline docs/guide/modules/review.md]: `cart` should be `review`

File: docs/guide/modules/review.md (hunk 2 above)
Comment (patzick) [inline docs/guide/modules/review.md]:
```suggestion
### Reviews
```
@degi just leave suggestions which can be commited into PR :)

File: docs/guide/modules/review.md (hunk 3 above)
Comment (patzick) [inline docs/guide/modules/review.md]:
```suggestion
The following events are published from `review` store:
```

**Comment by patzick** [comment]
@geektutor please commit these suggestions and we can merge that in :)

**Comment by patzick** [comment]
@geektutor are you willing to finish this?

**Comment by patzick** [comment]
closing due to long inactivity

**Pull request closed without merging**